### DISCUSSION TITLE
Can anyone share the Anura2023 installation files?

### DISCUSSION THREAD

**Discussion by HM-3214** [Anura3D/Anura3D_OpenSource#236] [Q&A] [answered]
Dear all,

I met exactly the same issue while compiling the latest Anura2024 as: #235 
While waiting for any solution, can anyone share the Anura2023 installation files so that I can install the previous version first?
Thanks a lot.

**Comment by lurpi** [comment, marked as answer]
In github, you can get any previous version of the same repository, you need to go through the commits ( https://github.com/Anura3D/Anura3D_OpenSource/commits )
As far as I could find, the official release of 2023 is here:
https://github.com/Anura3D/Anura3D_OpenSource/tree/539678b0fb157d5c322120c2c375330f2347d972

**Comment by coarse-snow** [comment]
hello,@HM-3214, I met the same problem , have you successfully compiled the 2023 version? thanks
